### PR TITLE
Add Play 2.5 support

### PR DESCRIPTION
**Pull request by fiadliel**

### CODE REVIEW

File: yax/play/src/main/scala/org/lyranthe/prometheus/client/integration/play/filters/PrometheusFilter.scala [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
 @Singleton
-class PrometheusFilter @Inject()(registry: Registry,
-                                 executionContext: ExecutionContext)
-    extends Filter {
-
-  private implicit val defaultRegistry = registry
-
-  private implicit val localExecutionContext = executionContext
-
+class PrometheusFilter @Inject()(implicit
```
Comment (fiadliel) [inline yax/play/src/main/scala/org/lyranthe/prometheus/client/integration/play/filters/PrometheusFilter.scala:16]: Yes, this walks back the implicit removal :-)
Trying to follow what the documentation does, and Play 2.5 docs use implicits.

File: project/yax.scala [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      }
+    }
+
+  // TODO obey [[sbt.Keys.crossPaths]] setting
+  private def srcDirs(root: File, config: String, sbv: String) =
+    List("", s"-$sbv").map { postfix =>
+      root / s"/src/$config/scala$postfix"
+    }
+
+  private def foo(root: File, config: String, flags: String*) = Def.task {
```
Comment (nemo83) [inline project/yax.scala:73]: maybe rename `foo` in something more meaningful

File: project/yax.scala [hunk 3]
```diff
@@ -0,0 +1,93 @@
+import sbt._
+import sbt.Keys._
+import java.io._
+import scala.io.Source
+
+object yax {
+
+  private def process(file: File, lines: List[String], flags: Set[String]): List[String] = {
```
Comment (nemo83) [inline project/yax.scala:8]: what does yax stand for?

File: yax/play/src/main/scala/org/lyranthe/prometheus/client/integration/play/filters/PrometheusFilter.scala (hunk 1 above)
Comment (nemo83) [reply yax/play/src/main/scala/org/lyranthe/prometheus/client/integration/play/filters/PrometheusFilter.scala:16]: go for it!

**Review by nemo83 — APPROVED** [review]

File: project/yax.scala (hunk 2 above)
Comment (fiadliel) [reply project/yax.scala:73]: I'd like to keep it the same as "upstream". This is from the doobie project. Actually, do need to add this info to follow the licence (MIT)

File: project/yax.scala (hunk 3 above)
Comment (fiadliel) [reply project/yax.scala:8]: I forget the details, tpolecat would be the person to corner. Either about hacks or yaks. Definitely he was slightly horrified at having to write this kind of stuff to support both scalaz and cats.

Comment (nemo83) [reply project/yax.scala:8]: ok

**Pull request merged**